src/components/FeaturedProductsGrid.js

```diff
@@ -0,0 +1,39 @@
+import { useFetch } from "../utils/hooks/useFetch";
+import Card from "./common/Card";
+import Flex from "./common/StyledFlex";
+import ImageContainer from "./common/Image";
+
+const FeaturedProductGrid = () => {
+    const { data, isLoading } = useFetch('product',true);
```

Comment:
Remember to convert this component `stateless`. You need to move this fetch to the parent component and only pass the data as props.